docs/code.rst

```diff
@@ -179,6 +179,7 @@ Operators
 .. autoclass:: airflow.contrib.operators.spark_submit_operator.SparkSubmitOperator
 .. autoclass:: airflow.contrib.operators.sqoop_operator.SqoopOperator
 .. autoclass:: airflow.contrib.operators.ssh_operator.SSHOperator
+.. autoclass:: airflow.contrib.operators.winrm_operator.WinRMOperator
```

Comment:
This should be in alphabetic ordering... and V comes before W

Reply:
fixed, commit coming